view/jpa/src/main/kotlin/io/holunda/polyflow/view/jpa/data/ConverterExt.kt

```diff
@@ -192,3 +192,26 @@ fun DataEntryDeletedEvent.toEntity(
 }.apply {
   this.protocol = this.protocol.addModification(this, this@toEntity.deleteModification, this@toEntity.state)
 }
+
+/**
+ * Event to entity for an anonymization, if an optional entry exists.
+ */
+fun DataEntryAnonymizedEvent.toEntity(
+  revisionValue: RevisionValue,
+  oldEntry: DataEntryEntity,
+) = oldEntry.also {
+  it.protocol.forEach { protocolEntry ->
```

Comment:
Why not filtering here like below?

Reply:
because I want to keep all the protocol entries. I just want to change the username.